libs/Einsums/Config/include/Einsums/Config/CompilerSpecific.hpp

```diff
@@ -51,7 +51,8 @@
 #    define EINSUMS_DO_PRAGMA(X)                 _Pragma(#X)
 #    define EINSUMS_DISABLE_WARNING_PUSH         EINSUMS_DO_PRAGMA(GCC diagnostic push)
 #    define EINSUMS_DISABLE_WARNING_POP          EINSUMS_DO_PRAGMA(GCC diagnostic pop)
-#    define EINSUMS_DISABLE_WARNING(warningName) EINSUMS_DO_PRAGMA(GCC diagnostic ignored #warningName)
+#    define EINSUMS_DISABLE_WARNING_0(warningName) GCC diagnostic ignored warningName
+#    define EINSUMS_DISABLE_WARNING(warningName) EINSUMS_DO_PRAGMA(EINSUMS_DISABLE_WARNING_0(warningName))
```

Comment:
That explains why that never worked before.

Reply:
Tbh, not a big fan of how the preprocessor handles this kind of thing.